### ISSUE TITLE
Allow access to workflow/task inputs as well as outputs

### ISSUE THREAD

**Issue by TMiguelT** [openwdl/wdl#258]
It's a relatively common use-case where you want to be able to use the _inputs_ (not outputs) of a first task as the inputs for a second task. For instance, if you are aligning and then variant calling, both stages will need a reference_fasta parameter. It makes sense to say "use `alignment.reference_fasta` as the value for `variant_calling.reference_fasta`", because then you and the users of your workflow don't have to repeat themselves when creating the input JSON file.

I had hoped this would already worked, and tried the following code:

```wdl
task a {
  String in
  command {
    echo ${in}
  }
  output {
    File out = stdout()
  }
}

workflow wf {
  call a {
    input:
        in = "hi"
  }
  call a as a2 {
    input:
        in = a.in
  }
}
```

However WOMtool tells me:

```
ERROR: Call output not found: Call 'a' doesn't have an output 'in' (line 18, col 16).

        in = a.in
               ^
```

In other words, referring to `task.x` assumes that `x` is the *output* from the task. I think we should have some syntax for accessing inputs, such as `task.inputs.x`. 

Thoughts?

**Comment by DavyCats** [comment]
Why not just make a workflow input `reference_fasta` and pass that to both the alignment and variantcalling task? I think that would generally be the best practice if multiple calls take the same input.

eg.
```wdl
workflow wf {
  input {
    String in
  }
  call a {
    input:
        in = in
  }
  call a as a2 {
    input:
        in = in
  }
}

**Comment by TMiguelT** [comment]
Ah yes you're right, this is a fine solution
